README.md

```diff
@@ -95,4 +95,4 @@ You have two options:
 
 * Option 1. (recommended) Install wire-server on kubernetes using the configuration and instructions provided in [wire-server-deploy](https://github.com/wireapp/wire-server-deploy). This is the best option to run it on a server and recommended if you want to self-host wire-server.
 
-* Option 2. Compile everything in this repo, then you can use the `services/start-services-only.sh`. This option is intended as a way to try out wire-server on your local development machine and not suited for production.
+* Option 2. Compile everything in this repo, then you can use the `services/run-services`. This option is intended as a way to try out wire-server on your local development machine and not suited for production.
```

Comment:
Given that start-services-only.sh is only calling out to run-services, should that script be removed too?

Reply:
no, i'm worried somebody will still use it and stumble over the fact that it's gone.  the way i changed it it doesn't do anything, but it'll point everybody who uses it in the right direction.